.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Add common python & vscode stuff to the gitignore and don't include files like these